### PR TITLE
Sa/update docstrings

### PR DESCRIPTION
**Pull request by genestack-solomatin**
- fix links to python docs used in API reference
- break long lines
- small cleanups

### CODE REVIEW

File: docs/source/create_cla_docs.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+        .. autoattribute:: genestack_client.{name}.APPLICATION_ID
+
+'''
+
+import os
+print os.getcwd()
+
+
+def main():
+    with open('cla_reference._rst', 'w') as f:
```
Comment (daniel-ohayon) [inline docs/source/create_cla_docs.py]: why the ``._rst`` extension? 
If this was intentional, add ``cla_reference._rst`` to the ``.gitignore``

Comment (genestack-solomatin) [inline docs/source/create_cla_docs.py]: It was done to remove `WARNING: document isn't included in any toctree` but I found way to do it via `excludes` in `conf.py`

**Comment by daniel-ohayon** [comment]
the sphinx syntax for python classes is not used everywhere consistently in the docs.
For example, sometimes `argparse.ArgumentParser` is written as is, and sometimes it is written as `:py:class:`~argparse.ArgumentParser``

**Comment by daniel-ohayon** [comment]
Another docs-related issue is that the return type of `find_files_by_name` is documented as `IFile`, which is false.

**Comment by genestack-solomatin** [comment]
``:py:class:`~argparse.ArgumentParser` `` should be used inside text entries and  ``argparse.ArgumentParser`` for `:rtype:`

**Comment by genestack-solomatin** [comment]
TODO for next prs:
- merge https://github.com/genestack/python-client/tree/vb/update_documentation (after merge of this branch)
- add utils to api reference 
- rearrange scripts and docs in source folder and .gitignore files (make one special for docs folder)

**Pull request merged**